arlo_server/routes.py

```diff
@@ -360,15 +352,33 @@ def require_audit_admin_for_organization(organization_id: Optional[str]):
     )
 
 
+election_new_schema = {
+    "type": "object",
+    "properties": {
+        "auditName": {"type": "string"},
+        "organizationId": {"type": "string"},
+    },
+    "required": ["auditName"],
+}
+
+
 @app.route("/election/new", methods=["POST"])
 def election_new():
-    info = request.get_json()
+    election = request.get_json()
+    validate(election, election_new_schema)
 
-    organization_id = info.get("organizationId", None) if info else None
```

Comment:
I don't believe it's necessary anymore for two reasons:
- We validate that `info` (now `election`) against a schema that expects an object
- `info` should never be `None` since `auditName` is required